### PR TITLE
[Feature][Vince] update button links

### PR DESCRIPTION
**Pull request by VinceRaAle**
https://www.loom.com/share/f89501ab545c4a399872614aff0fdf9c

### CODE REVIEW

File: pages/admin.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import ComingSoon from "@/components/ComingSoon";
+import Layout from "@/components/Layout";
+import React from "react";
+
+function admin() {
```
Comment (maxellmilay) [inline pages/admin.tsx]: I pascal case ni hehe (e.g. PascalCase)

File: pages/give.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import ComingSoon from "@/components/ComingSoon";
+import Layout from "@/components/Layout";
+import React from "react";
+
+function give() {
```
Comment (maxellmilay) [inline pages/give.tsx]: casing issue

File: pages/ministries.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import ComingSoon from "@/components/ComingSoon";
+import Layout from "@/components/Layout";
+import React from "react";
+
+function ministries() {
```
Comment (maxellmilay) [inline pages/ministries.tsx]: casing issue

File: pages/sermons.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import ComingSoon from "@/components/ComingSoon";
+import Layout from "@/components/Layout";
+import React from "react";
+
+function sermons() {
```
Comment (maxellmilay) [inline pages/sermons.tsx]: casing issue

File: pages/index.tsx [hunk 5]
```diff
@@ -2,30 +2,39 @@ import { useEffect } from "react";
...
+        <a
+          href="https://www.facebook.com/tmifellowship/videos"
+          target="_blank"
+          rel="noreferrer"
+        >
           <button className="bg-transparent border border-white rounded-3xl py-1.5 px-7 text-xs sm:text-xl">
             See More Sermons
           </button>
-        </div>
+        </a>
```
Comment (maxellmilay) [inline pages/index.tsx]: use Link component

File: pages/index.tsx [hunk 6]
```diff
@@ -44,9 +53,33 @@ export default function Home() {
...
+                alt="Facebook"
+              />
+            </a>
+            <a
+              href="https://www.youtube.com/channel/UCXMRU2-nQZwMNQyjM64mLJA"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <SocialIcons imgURL="/images/socials/youtube.png" alt="Youtube" />
+            </a>
```
Comment (maxellmilay) [inline pages/index.tsx:85]: use Link nextjs component if possible

Comment (ezerssss) [reply pages/index.tsx:85]: diba diay ang Link nga component kay gamiton ra for web pages nga ga exists sa web app?

File: pages/admin.tsx (hunk 1 above)
Comment (ezerssss) [reply pages/admin.tsx]: resolved

File: pages/give.tsx (hunk 2 above)
Comment (ezerssss) [reply pages/give.tsx]: resolved

File: pages/ministries.tsx (hunk 3 above)
Comment (ezerssss) [reply pages/ministries.tsx]: resolved

File: pages/sermons.tsx (hunk 4 above)
Comment (VinceRaAle) [reply pages/sermons.tsx]: resolved

**Review by ezerssss — APPROVED** [review]

File: pages/index.tsx (hunk 6 above)
Comment (maxellmilay) [reply pages/index.tsx:85]: oh okok ana diay na, mbmb

**Review by maxellmilay — APPROVED** [review]

**Pull request merged**